optuna/pruners/base.py

```diff
@@ -30,3 +31,26 @@ def prune(self, study, trial):
         """
 
         raise NotImplementedError
+
+    @abc.abstractmethod
+    def get_trial_pruner_auxiliary_data(self, study_id, trial_number):
+        # type: (int, int) -> str
```

Comment:
Reflecting the above change.
```suggestion
        # type: (str, int) -> str
```